src/main/java/org/opensrp/service/PractitionerService.java

```diff
@@ -113,6 +113,20 @@ public void deletePractitioner(String identifier) {
 		
 	}
 
+	@PreAuthorize("hasRole('PRACTITIONER_VIEW')")
```

Comment:
Update the permissions and roles check to the following

```
@PreAuthorize("hasRole('ORGANIZATION_VIEW')")
@PostFilter("hasPermission(filterObject, 'ORGANIZATION_VIEW')")
```

Reply:
Done